pandas/core/indexes/datetimes.py

```diff
@@ -1968,14 +1973,45 @@ def tz_localize(self, tz, ambiguous='raise', errors='raise'):
             .. deprecated:: 0.15.0
                Attempt to infer fall dst-transition hours based on order
 
+
         Returns
         -------
-        localized : DatetimeIndex
+        DatetimeIndex
+
+        Examples
+        --------
+        In the example below, We put the date range from 01 March 2018
+        to 08 March 2018 & localize this to US/Eastern Time zone & again
+        we perform reverse operation where we remove tize zone & make it
+        tz-naive
+
+        >>> dti = pd.date_range('2018-03-01', '2018-03-03')
+
+        >>> dti
+        DatetimeIndex(['2018-03-01', '2018-03-02', '2018-03-03'],
+        dtype='datetime64[ns]', freq='D')
+
+        #localize DatetimeIndex in US?Eastern time zone.
+        >>> tz_aware = dti.tz_localize(tz='US/Eastern')
+
+        >>> tz_aware
+        DatetimeIndex(['2018-03-01 00:00:00-05:00',
+        '2018-03-02 00:00:00-05:00', '2018-03-03 00:00:00-05:00'],
+        dtype='datetime64[ns, US/Eastern]', freq='D')
+
+        #localize aware time zone into naive time zone.
```

Comment:
Sorry, that was my fault, but when I said to add comments to the examples, I didn't mean Python comments, but just the sentence. So, without the `#`, in capital letters, and with a blank line after it (also the one before you already have).

Reply:
sure